packages/fxa-content-server/app/scripts/lib/constants.js

```diff
@@ -86,6 +86,9 @@ module.exports = {
   PROFILE_IMAGE_MIN_WIDTH: 100,
   DEFAULT_PROFILE_IMAGE_MIME_TYPE: 'image/jpeg',
 
+  // Limit to 20 megabytes
+  PROFILE_FILE_IMAGE_MAX_UPLOAD_SIZE: 20 * 1000 * 1000,
```

Comment:
I thought the value for this was going to be [this](https://github.com/mozilla/fxa/blob/main/packages/fxa-profile-server/lib/config.js#L100-L104), but it looks like in production it's actually `1024 * 1024 * 1.35`

Reply:
I just picked an arbitrary value to be the limit. What number do you think it's better? Should I choose `1.35*1024*1024`?

Reply:
Yes I believe so, given that’s what we have for the server-side validation.

Reply:
If there is a server-side validation, then why it is possible to upload a big image?

Reply:
I believe that's where folks are seeing the "System unavailable, try again soon" error message. But it's possible something else is going on that I am unaware of. 

Thank you for updating! I've verified locally that a 3MB file fails, but a 1.2MB file is accepted, as well as some other sizes to be sure.